src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx

```diff
@@ -0,0 +1,58 @@
+
+## Charts [#charts]
+
+<img
+  title="nrql-chart-example.png"
+  alt="A screenshot example of a chart"
+  src={chartsChartUi}
+/>
+<figcaption>
+  This is a chart created by a NRQL query showing the total number of transactions for each reported application across a period of 1 hour.
+</figcaption>
+
+Configuring a chart to represent your data is often the last step of creating a NRQL query. New Relic offers [multiple options](/docs/query-your-data/explore-query-data/use-charts/chart-types) to help you find the perfect way to visualize your data, and you can do many things with your charts once you've made them. You can:
+
+* Use your chart as the start of a [new dashboard](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards#dashboards-create) to monitor multiple data streams from a single location
+* [Add your chart to an existing dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#add-pages-edit) to add another data stream to your monitoring
+* [Share your chart](/docs/query-your-data/explore-query-data/use-charts/use-your-charts#share-charts) with others by copying the permalink or downloading it as a PDF (or CSV, if applicable)
+* Edit the NRQL query behind the chart to help fine-tune the data you're seeing
+* [Create an alert](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition#create-simple-alert) from the chart to help proactively notify you when problems arise
+
+Charts are great on their own, but they really shine when you compile them together to create dashboards.
+
+## Dashboards [#dashboards]
+
+<img
+  title="nrql-dashboard-example.png"
+  alt="A screenshot example of a dashboard"
+  src={dashboardsDashboardUi}
+/>
+<figcaption>
+  A dashboard representing data in pie and bar chart formats.
+</figcaption>
+
+While charts are great for reporting on individual pieces, dashboards are the best way to get a much broader view of your data: if charts are pieces to a puzzle, dashboards are the puzzle completed. Dashboards are highly configurable, and you can use them to show you as many or as few data streams as you like. You can:
+
+* [Add charts](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#dash-create-chart) (using the query builder) or your own [text, images, or links](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#markdown) (using Markdown) to your dashboard
```

Comment:
@jeff-colucci this was a bit tough for me to parse with the parenthetical comments. How about just making separate bullets?

* [Add charts](https://docswebsitedevelop-nrqlintrochartsdashboards.gatsbyjs.io/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#dash-create-chart) to your dashboard using the query builder
* Add your own text, images, or links](https://docswebsitedevelop-nrqlintrochartsdashboards.gatsbyjs.io/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#markdown) using Markdown

Reply:
Good call, changes have been made!